src/scripting/game_lua_kernel.cpp

```diff
@@ -2750,6 +2750,44 @@ int game_lua_kernel::intf_set_floating_label(lua_State* L, bool spawn)
 	return 1;
 }
 
+namespace
+{
+	const unit_map& get_unit_map()
+	{
+		// Used if we're in the game, including during the construction of the display_context
+		if(resources::gameboard) {
+			return resources::gameboard->units();
+		}
+
+		// If we get here, we're in the scenario editor
+		assert(display::get_singleton());
+		return display::get_singleton()->context().units();
+	}
+	void reset_affect_adjacent(const unit& u_)
+	{
+		bool affect_adjacent = false;
+		for(const auto [key, cfg] : u_.abilities().all_children_view()) {
+			bool image_or_hides = (key == "hides" || cfg.has_attribute("halo_image") || cfg.has_attribute("overlay_image"));
+			if(image_or_hides && cfg.has_child("affect_adjacent")){
+				affect_adjacent = true;
+				break;
+			}
+		}
+		const unit_map& units = get_unit_map();
+		if(affect_adjacent){
```

Comment:
Space between `if` and `condition` as well between `)` and `{` please. Same for other cases.
```suggestion
		if (affect_adjacent) {
```